backend/schedulerd/interface.go

```diff
@@ -0,0 +1,56 @@
+package schedulerd
+
+import corev2 "github.com/sensu/sensu-go/api/core/v2"
+
+// Scheduler is the abstract interface of a scheduler.
+type Scheduler interface {
```

Comment:
This pleases me.

Reply:
Ah that reminds me I have to improve the docs for that type. "Scheduler is the abstract interface of a scheduler" is not great documentation. :)